File: android/src/main/java/pers/hal42/util/Storable.java
package pers.hal42.util;
import java.lang.annotation.*;

@Retention(RetentionPolicy.RUNTIME)
@Target(ElementType.FIELD)
@interface Storable {
  //marker for field to be asciified and transported
  String key() default "";//empty string results in Field's name being used as the key.
}

File: android/src/main/kotlin/pers/hal42/util/Storable.kt
package pers.hal42.util

//kotlin default: @Retention(RetentionPolicy.RUNTIME)
@Target(AnnotationTarget.FIELD)
annotation class Storable(//marker for field to be asciified and transported
  val key: String = ""//empty string results in Field's name being used as the key.
)
